src/main/java/carpet/mixins/WorldRenderer_scarpetRenderMixin.java

```diff
@@ -2,39 +2,39 @@
 
 import carpet.network.CarpetClient;
 import carpet.script.utils.ShapesRenderer;
-import net.minecraft.client.MinecraftClient;
-import net.minecraft.client.render.BufferBuilderStorage;
-import net.minecraft.client.render.Camera;
-import net.minecraft.client.render.GameRenderer;
-import net.minecraft.client.render.LightmapTextureManager;
-import net.minecraft.client.render.WorldRenderer;
-import net.minecraft.client.util.math.MatrixStack;
-import net.minecraft.util.math.Matrix4f;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.math.Matrix4f;
+import net.minecraft.client.Camera;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.client.renderer.LevelRenderer;
+import net.minecraft.client.renderer.LightTexture;
+import net.minecraft.client.renderer.RenderBuffers;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
-@Mixin(WorldRenderer.class)
+@Mixin(LevelRenderer.class)
 public class WorldRenderer_scarpetRenderMixin
 {
     @Inject(method = "<init>", at = @At("RETURN"))
-    private void addRenderers(MinecraftClient client, BufferBuilderStorage bufferBuilders, CallbackInfo ci)
+    private void addRenderers(Minecraft client, RenderBuffers bufferBuilders, CallbackInfo ci)
     {
         CarpetClient.shapes = new ShapesRenderer(client);
     }
 
-    @Inject(method = "render", at =  @At(
+    @Inject(method = "renderLevel", at =  @At(
             value = "INVOKE",
-            target = "Lnet/minecraft/client/render/TexturedRenderLayers;getEntityTranslucentCull()Lnet/minecraft/client/render/RenderLayer;", shift = At.Shift.BEFORE
+            target = "Lnet/minecraft/client/renderer/Sheets;translucentCullBlockSheet()Lnet/minecraft/client/renderer/RenderType;", shift = At.Shift.BEFORE
             //target = "Lnet/minecraft/client/render/RenderLayer;getWaterMask()Lnet/minecraft/client/render/RenderLayer;", shift = At.Shift.AFTER
             //target = "Lnet/minecraft/client/render/BufferBuilderStorage;getEffectVertexConsumers()Lnet/minecraft/client/render/VertexConsumerProvider$Immediate;", shift = At.Shift.BEFORE
```

Comment:
I wish it renamed in the comments as well, but hey, you get what you get

Reply:
Can be done manually :)

Reply:
Though there's too many mixins too check manually, so it can be done when needed via fabric bot (on DMs, after running `!setNamespace`) or via Linkie bot, doesn't need DMs (but also works with them).